rero_ils/modules/documents/views.py

```diff
@@ -323,3 +325,44 @@ def item_status_text(item, format='medium', locale='en'):
             if item.status == ItemStatus.IN_TRANSIT:
                 text += ' ({0})'.format(_(ItemStatus.IN_TRANSIT))
     return text
+
+
+@blueprint.app_template_filter()
+def identifiedby_format(identifiedby):
+    """Format identifiedby for template."""
+    output = []
+    for identifier in identifiedby:
+        status = identifier.get('status')
+        id_type = identifier.get('type')
+        if status not in ('invalid', 'cancelled', 'invalid or cancelled') and \
+                id_type != 'bf:Local':
+            if id_type.startswith('bf:'):
+                id_type = id_type[3:]
+            output.append({'type': id_type, 'value': identifier.get('value')})
+    return output
+
+
+@blueprint.app_template_filter()
+def document_isbn(identifiedby):
+    """Returns document isbn."""
+    for identifier in identifiedby:
+        if identifier.get('type') == 'bf:Isbn':
+            return {'isbn': identifier.get('value')}
+    return {}
+
+
+@blueprint.app_template_filter()
+def language_format(langs_list, interface_language):
+    """Format language for template."""
+    output = []
+    for lang in langs_list:
+        try:
+            lang_name = pycountry.languages.get(
+                bibliographic=lang.get('value')).alpha_2
+        except:
+            lang_name = pycountry.languages.get(
+                alpha_3=lang.get('value')).alpha_2
+        lang_display = Locale(lang_name).get_language_name(
+            interface_language).capitalize()
```

Comment:
In French the second word after the comma has no capital letters on the first letter. Look at the rules for the other languages.

Reply:
all languages are now displayed in lower case.